app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -82,8 +82,107 @@ class CourseSerializer < ActiveModel::Serializer
     attribute(:ProviderLocation)                      { provider_location }
     attribute(:ContactDetails)                        { contact_details }
 
+    # DescriptionSections_Mapping
+    attribute(:DescriptionSections)                   { description_sections }
+
   private
 
+    def default_description_section_value
+      {
+        Id: 0,
+        Ordinal: 0,
+        CourseId: 0,
+        Course: nil,
+      }
+    end
+
+    def text(enrichment_type, enrichment_key)
```

Comment:
Can we call this `lookup_description_text` or something descriptive like this?

Reply:
drop in flavour of inliner